lxd/db/cluster/open.go

```diff
@@ -289,18 +278,16 @@ func checkClusterIsUpgradable(ctx context.Context, tx *sql.Tx, target [2]int) er
 			// Our version is bigger, we should stop here
 			// and wait for other nodes to be upgraded and
 			// restarted.
-			return errSomeNodesAreBehind
+			return api.StatusErrorf(http.StatusPreconditionFailed, "A cluster member's version (%v) is behind this cluster member's version (%v), please ensure versions match", version, target)
 		case 2:
 			// Another node has a version greater than ours
-			// and presumeably is waiting for other nodes
+			// and presumably is waiting for other nodes
 			// to upgrade. Let's error out and shutdown
 			// since we need a greater version.
```

Comment:
I don't think this comment is accurate anymore, since LXD now waits for an upgrade notification in this case as well.

Reply:
Ah yeah ill do a follow up pr

Reply:
https://github.com/canonical/lxd/pull/15322